# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/e3/ln-sf.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Allow re-installing over existing e3 symlinks.

diff -Nurp e3editor.orig/Makefile e3editor/Makefile
--- e3editor.orig/Makefile	2026-04-24 14:22:57.000000000 +0200
+++ e3editor/Makefile	2026-09-27 18:21:15.360858933 +0200
@@ -65,11 +65,11 @@ endif
 install:	e3
 	install -d $(DESTDIR)$(PREFIX) $(DESTDIR)$(BINDIR) $(DESTDIR)$(MANDIR)
 	install -m 755 e3 $(DESTDIR)$(BINDIR)
-	cd $(DESTDIR)$(BINDIR) && ln -s e3 e3ws
-	cd $(DESTDIR)$(BINDIR) && ln -s e3 e3em
-	cd $(DESTDIR)$(BINDIR) && ln -s e3 e3pi
-	cd $(DESTDIR)$(BINDIR) && ln -s e3 e3vi
-	cd $(DESTDIR)$(BINDIR) && ln -s e3 e3ne
+	cd $(DESTDIR)$(BINDIR) && ln -sf e3 e3ws
+	cd $(DESTDIR)$(BINDIR) && ln -sf e3 e3em
+	cd $(DESTDIR)$(BINDIR) && ln -sf e3 e3pi
+	cd $(DESTDIR)$(BINDIR) && ln -sf e3 e3vi
+	cd $(DESTDIR)$(BINDIR) && ln -sf e3 e3ne
 	install -m 644 e3.man $(DESTDIR)$(MANDIR)/e3.$(MANSEC)
 
 clean:
